quinn/Cargo.toml

```diff
@@ -46,6 +46,7 @@ thiserror = "1.0.21"
 tracing = "0.1.10"
 tokio = { version = "1.28.1", features = ["sync"] }
 udp = { package = "quinn-udp", path = "../quinn-udp", version = "0.5", default-features = false }
+atomic-waker = "1.1.2"
```

Comment:
Nit: alphabetic order?

Reply:
Oops, I thought `cargo add` would preserve that.